File: ServerSide_servlet/src/java/edit.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/JSP_Servlet/Servlet.java to edit this template
 */

import java.io.IOException;
import java.io.PrintWriter;
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.ResultSet;
import java.sql.SQLException;
import java.sql.Statement;
import java.util.logging.Level;
import java.util.logging.Logger;
import javax.servlet.ServletException;
import javax.servlet.annotation.WebServlet;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

/**
 *
 * @author Waseem
 */
@WebServlet(urlPatterns = {"/edit"})
public class edit extends HttpServlet {

    /**
     * Processes requests for both HTTP <code>GET</code> and <code>POST</code>
     * methods.
     *
     * @param request servlet request
     * @param response servlet response
     * @throws ServletException if a servlet-specific error occurs
     * @throws IOException if an I/O error occurs
     */
    protected void processRequest(HttpServletRequest request, HttpServletResponse response)
            throws ServletException, IOException, ClassNotFoundException, SQLException {
        response.setContentType("text/html;charset=UTF-8");
        try (PrintWriter out = response.getWriter()) {
            /* TODO output your page here. You may use following sample code. */
            Class.forName("com.mysql.jdbc.Driver");
            try (Connection c = DriverManager.getConnection("jdbc:mysql://localhost:3306/network2", "root", "")) {
                Statement s = c.createStatement();
                String Fname = request.getParameter("fname");
                String Lname = request.getParameter("lname");
                String name = Fname + " " + Lname;
                String Address = request.getParameter("address");
                String Email = request.getParameter("email");
                String Pass = request.getParameter("pass");
                int Id = Integer.parseInt(request.getParameter("id"));
                String Photo = request.getParameter("img");
                int email_ex = 0;
                ResultSet sqll = s.executeQuery("SELECT `email` FROM `employee` WHERE `id` != '" + Id + "'");
                String sql = "UPDATE `employee` SET `name`='" + name + "',`address`='" + Address + "',`email`= '" + Email + "',`password`= '" + Pass + "', `photo`='" + Photo + "' WHERE `id` = '" + Id + "'";
                while (sqll.next()) {
                    if (sqll.getString("email") == Email) {
                        email_ex = 1;
                    }
                }
                if (email_ex == 1) {
                    out.print("Email is already exist!");
                } else {
                    int sr = s.executeUpdate(sql);
                    if (sr > 0) {
                        out.print("User Updated successfully");
                    } else {
                        out.print("No user with this Id or Name");
                    }
                }
            }
        }
    }

    // <editor-fold defaultstate="collapsed" desc="HttpServlet methods. Click on the + sign on the left to edit the code.">
    /**
     * Handles the HTTP <code>GET</code> method.
     *
     * @param request servlet request
     * @param response servlet response
     * @throws ServletException if a servlet-specific error occurs
     * @throws IOException if an I/O error occurs
     */
    @Override
    protected void doGet(HttpServletRequest request, HttpServletResponse response)
            throws ServletException, IOException {
        try {
            processRequest(request, response);
        } catch (ClassNotFoundException ex) {
            Logger.getLogger(edit.class.getName()).log(Level.SEVERE, null, ex);
        } catch (SQLException ex) {
            Logger.getLogger(edit.class.getName()).log(Level.SEVERE, null, ex);
        }
    }

    /**
     * Handles the HTTP <code>POST</code> method.
     *
     * @param request servlet request
     * @param response servlet response
     * @throws ServletException if a servlet-specific error occurs
     * @throws IOException if an I/O error occurs
     */
    @Override
    protected void doPost(HttpServletRequest request, HttpServletResponse response)
            throws ServletException, IOException {
        try {
            processRequest(request, response);
        } catch (ClassNotFoundException ex) {
            Logger.getLogger(edit.class.getName()).log(Level.SEVERE, null, ex);
        } catch (SQLException ex) {
            Logger.getLogger(edit.class.getName()).log(Level.SEVERE, null, ex);
        }
    }

    /**
     * Returns a short description of the servlet.
     *
     * @return a String containing servlet description
     */
    @Override
    public String getServletInfo() {
        return "Short description";
    }// </editor-fold>

}

File: ServerSide_PHP/edit.php
<?php
$servername = "localhost";
$username = "root";
$password = "";

// Create connection
$conn = new mysqli($servername, $username, $password);
$db = new mysqli('localhost', 'root', '', 'network2');

// Check connection
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}

if (isset( $_REQUEST['name'])&& isset( $_REQUEST['Add'])&& isset( $_REQUEST['Email'])
    && isset( $_REQUEST['Pass'])&& isset( $_REQUEST['Id']) && isset( $_REQUEST['img'])) {
    $Fname = $_REQUEST['Fname'];
    $Lname = $_REQUEST['Lname'];

    $Name=  $_REQUEST['name'];
    $Address = $_REQUEST['Add'];
    $Email = $_REQUEST['Email'];
    $Pass = $_REQUEST['Pass'];
    $Id = (int)$_REQUEST['Id'];
    $Photo = $_REQUEST['img'];
    $email_ex = 0;
    $sqll = "SELECT email FROM network2.employee WHERE id != '$Id'";
    $resultt = $conn->query($sqll);
    $sql = "UPDATE network2.employee SET name='$Name',address= '$Address',email= '$Email',password= '$Pass', photo='$Photo' WHERE id = '$Id'";

    if ($resultt->num_rows > 0) {
        while($row = $resultt->fetch_assoc()) {
            if ($row["email"] == $Email){
                $email_ex = 1;
            }
        }
        if ($email_ex == 1) {
            echo "Email is already exist!";
        }
        else {
            $result = $conn->query($sql);
            echo "User Updated successfully";
        }
    }
}
?>